src/Language/PureScript/Docs/Render.hs

```diff
@@ -152,9 +152,14 @@ renderDeclaration (P.DataDeclaration dtype name args ctors) title =
   children = map renderCtor ctors
   -- TODO: Comments for data constructors?
   renderCtor (ctor', tys) =
-          let typeApp' = foldl P.TypeApp (P.TypeConstructor (P.Qualified Nothing ctor')) tys
-              childCode = renderType typeApp'
-          in  RenderedChildDeclaration (show ctor') Nothing childCode Nothing ChildDataConstructor
+    RenderedChildDeclaration (show ctor') Nothing Nothing (ChildDataConstructor ctorSignature ctorCode)
+    where
+    ctor'' = P.TypeConstructor (P.Qualified Nothing ctor')
+    typeApp' = foldl P.TypeApp ctor'' tys
+    ctorSignature = renderType typeApp'
+    -- TODO: add forall for all free type variables
```

Comment:
not any more, the code two lines below this comment does this. whoops! will fix.